### PR TITLE
Create and continue registration

### PR DESCRIPTION
**Pull request by LayCraft**
@asanchezr 

Work on the form. Lots of data massaging.

### CODE REVIEW

File: embc-app/ClientApp/src/app/volunteer-dashboard/volunteer-dashboard.component.html [hunk 1]
```diff
@@ -72,7 +72,7 @@ <h3 class="blue-header">LIST OF EVACUEES</h3>
   <div class="col-12 col-lg-4">
     <app-side-box-info title="ADD AN EVACUEE"
       content="Before adding volunteers, confirm that they have been registered by the Emergency Social Services Director."
```
Comment (asanchezr) [inline embc-app/ClientApp/src/app/volunteer-dashboard/volunteer-dashboard.component.html]: Please check the content on the wireframes. The link to add evacuees and content of the box ("add more volunteers") do not match.

File: embc-app/ClientApp/src/app/store/registration/registration.reducer.ts [hunk 2]
```diff
@@ -51,6 +51,7 @@ const initialState: State = {
     hasPersonalServicesReferral: null,
     hasChildCareReferral: null,
     hasPetCareReferral: null,
+    referralOther: null,
```
Comment (asanchezr) [inline embc-app/ClientApp/src/app/store/registration/registration.reducer.ts]: This field is not needed. We already modelled it under the name **externalReferralsDetails** 
Please see Line 45 above

File: embc-app/ClientApp/src/app/evacuee-registration/evacuee-registration.component.ts [hunk 3]
```diff
@@ -86,38 +137,39 @@ export class EvacueeRegistrationComponent implements OnInit {
       return null;
     }
   }
+
   initForm(): void {
     this.form = this.formBuilder.group({
-      restrictedAccess: null,
+      restrictedAccess: true,
```
Comment (asanchezr) [inline embc-app/ClientApp/src/app/evacuee-registration/evacuee-registration.component.ts]: Why setting this to **true** by default, should be left null so the volunteer makes a choice

Comment (LayCraft) [reply embc-app/ClientApp/src/app/evacuee-registration/evacuee-registration.component.ts]: This was for debugging the form. I know that I had an error dominating my browser console regarding a missing value for this.

File: embc-app/ClientApp/src/app/store/registration/registration.reducer.ts (hunk 2 above)
Comment (LayCraft) [reply embc-app/ClientApp/src/app/store/registration/registration.reducer.ts]: Ok. I will change this.

File: embc-app/ClientApp/src/app/volunteer-dashboard/volunteer-dashboard.component.html (hunk 1 above)
Comment (LayCraft) [reply embc-app/ClientApp/src/app/volunteer-dashboard/volunteer-dashboard.component.html]: Ah yes. I had built both boxes and intended to comment one out. I will fix this in a bit.

**Pull request merged**